.github/workflows/deploy-code.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7]
+        python-version: [3.8]
```

Comment:
I am not sure about changing this at present. I do not think it really matters but since we still will be producing releases for 3.7, including this imminent one, maybe its better to leave it at 3.7 until we drop that and 3.8 becomes the min level we support.

Reply:
Good point. I should revert this.